src/Raven.Server/Program.cs

```diff
@@ -202,15 +202,23 @@ public static int Main(string[] args)
                                     "For more information go to https://ravendb.net/l/EJS81M/4.1";
                             }else if (e is SocketException && PlatformDetails.RunningOnPosix)
                             {
-                                message +=
+                                message =
                                     $"{Environment.NewLine}In Linux low-level port (below 1024) will need a special permission, if this is your case please run{Environment.NewLine}" +
                                     $"sudo setcap CAP_NET_BIND_SERVICE=+eip {typeof(RavenServer).Assembly.Location}";
                             }
 
                             if (Logger.IsOperationsEnabled)
+                            {
                                 Logger.Operations("Failed to initialize the server", e);
+                                Logger.Operations(message);
+                            }
+
                             Console.WriteLine(message);
 
+                            Console.WriteLine();
+
+                            Console.WriteLine(e);
```

Comment:
This line reverts assumption from http://issues.hibernatingrhinos.com/issue/RavenDB-11107

Reply:
maybe wrap this in `#if DEBUG` ?

Reply:
We need this to get a clear error for _us_ in prod.
When a user runs into a problem, we need to have the full details.
You can see that the message is still shown first, along with the full error stack that we can use.

Reply:
Right now, see what happens when you configure a bad certificate and start the server.